source/system/String.ooc

```diff
@@ -16,7 +16,7 @@ String: class {
 	// Avoid direct buffer access, as it breaks immutability.
 	_buffer: CharBuffer
 	size ::= this _buffer size
-
+	_isLiteral := false
```

Comment:
It is null if third argument to `CharBuffer` constructor is `true`, which seems to be the case only in `makeStringLiteral`.